src/main/java/org/springframework/data/elasticsearch/core/ElasticsearchTemplate.java

```diff
@@ -136,7 +137,8 @@
  */
 public class ElasticsearchTemplate implements ElasticsearchOperations, ApplicationContextAware {
 
-	private static final Logger logger = LoggerFactory.getLogger(ElasticsearchTemplate.class);
+	private static final Logger QUERY_LOGGER = LoggerFactory.getLogger("org.springframework.data.elasticsearch.core.QUERY");
```

Comment:
I like this idea, though I don't know if there are any conventions within Spring / Spring Data to look towards following for this type of logging?  For example in the JPA project they have the following options for logging queries - http://www.baeldung.com/sql-logging-spring-boot